docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -748,6 +750,41 @@ In this example, the table is partitioned by the month of ``order_date``, a hash
         country VARCHAR)
     WITH (partitioning = ARRAY['month(order_date)', 'bucket(account_number, 10)', 'country'])
 
+Sorted tables
+------------------
+
+In addition to partitioning, Iceberg tables may have a specified list of sort columns. When writes
+are made to the table, each file will individually be sorted by these columns. Sorting can be
+particularly helpful when selective reads are frequently made against a column with a high cardinality.
```

Comment:
"against" or sth like "with predicates on [columns table is sorted on]" ?
(my wording isn't good, that's just for conveying the intention)